### PR TITLE
added PR template

### PR DESCRIPTION
**Pull request by will0684**
Added a template for PRs that should cover our needs.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Acceptance Criteria
+
+Get the acceptance criteria for the story and/or task you are working on and put it all here.
+
+## Test Instructions
+
+1. Steps on...
+1. how to test...
+1. or inspect this...
+1. feature manually.
```
Comment (pcraig3) [inline .github/pull_request_template.md:16]: This is probably a 1., 2., 3., 4. list, eh?

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,3231 @@
 {
   "name": "alpha-site",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
Comment (pcraig3) [inline package-lock.json:6]: Not sure why this keeps happening to your PRs, @will0684. The actual PR template looks fine, but we definitely don't want to include 3000+ lines in the package-lock file as we didn't add any new packages here.

**Review by pcraig3 — CHANGES_REQUESTED** [review]
Made a couple comments. Mostly, we should remove the package-lock stuff.

File: .github/pull_request_template.md (hunk 1 above)
Comment (will0684) [reply .github/pull_request_template.md:16]: You are correct, fixing now.

File: package-lock.json (hunk 2 above)
Comment (will0684) [reply package-lock.json:6]: I agree. I'll try to figure out what's going on here.

Comment (Moro-Code) [reply package-lock.json]: could you be using node 14 as opposed to 12 ?

Comment (will0684) [reply package-lock.json]: Yes I am using 15 it seems, I'll switch down to 12 for the time being.

File: .github/pull_request_template.md (hunk 1 above)
Comment (will0684) [reply .github/pull_request_template.md]: Fixed

**Review by pcraig3 — APPROVED** [review]
Looks good! We could also look at putting in a french translation for this if any devs would like it.

**Pull request merged**